internal/cli/cmd/bench/ycsb.go

```diff
@@ -190,7 +191,9 @@ func (o *YcsbOptions) Validate() error {
 		}
 	}
 	if !supported {
-		return fmt.Errorf("driver %s is not supported", o.Driver)
+		return fmt.Errorf("ycsb now only support driver in %s, your cluster driver is %s"+
```

Comment:
support driver -> supports drivers; your cluster driver is -> current cluster driver is; remove "if your cluster belongs to the category of one of them, please set the driver to one of them"